docs/api/cluster.md

```diff
@@ -38,11 +38,20 @@ Returns general information about the clusters managed by the YuniKorn Scheduler
 
 **Content examples**
 
-As an example, here is a response from a 2-node cluster with 3 applications and 4 running containers.
+As an example, here is a response from a 2-node cluster with 3 applications and 4 running containers and 1 resource manager.
 
 ```json
 [
     {
+        "scheduleStartDate": "2022-02-21T11:12:05+0000",
+        "rmBuildInformation": [
+          {
+            "buildDate": "2022-02-21T19:09:16+0800",
+            "buildVersion": "latest",
+            "isPluginVersion": "false",
+            "rmId": "rm-123"
+          }
```

Comment:
Fix the indent please: brackets are indented 4 spaces, the content is indented another 4 spaces

Reply:
Modified. Thanks.